chrome/script.js

```diff
@@ -9,7 +9,7 @@ document.addEventListener("blue-blocker-event", function (e) {
 		const body = JSON.parse(e.detail.body);
 
 		switch (e.detail.parsedUrl[1]) {
-			case "HomeLatestTimeline":
+			case "HomeTimeline":
```

Comment:
a quick check on twitter shows that the for you tab uses HomeTimeline and following uses HomeLatestTimeline. you can change your PR so that both are included or I'll update whenever I get around to it